packages/transport/src/transports/abstractApi.ts

```diff
@@ -174,20 +174,26 @@ export abstract class AbstractApiTransport extends AbstractTransport {
     }: AbstractTransportMethodParams<'call'>) {
         return this.scheduleAction(
             async signal => {
-                const getPathBySessionResponse = await this.sessionsClient.getPathBySession({
-                    session,
-                });
-                if (!getPathBySessionResponse.success) {
-                    // session not found means that device was disconnected
-                    if (getPathBySessionResponse.error === 'session not found') {
-                        return this.error({ error: ERRORS.DEVICE_DISCONNECTED_DURING_ACTION });
+                const handleError = (error: string) => {
+                    // if user revokes usb permissions in browser we need a way how propagate that the device was technically disconnected,
+                    if (error === ERRORS.DEVICE_DISCONNECTED_DURING_ACTION) {
+                        this.enumerate();
```

Comment:
note: this is not new, just moved. but it should be moved to a higher layer - connect